pkgs/applications/editors/sublime/4/common.nix

```diff
@@ -9,9 +9,10 @@
 let
   pnameBase = "sublimetext4";
   packageAttribute = "sublime4${lib.optionalString dev "-dev"}";
-  binaries = [ "sublime_text" "plugin_host-3.3" "plugin_host-3.8" "crash_reporter" ];
+  binaries = [ "sublime_text" "plugin_host-3.3" "plugin_host-3.8" crashHandlerBinary ];
   primaryBinary = "sublime_text";
   primaryBinaryAliases = [ "subl" "sublime" "sublime4" ];
+  crashHandlerBinary = if dev then "crash_handler" else "crash_reporter";
```

Comment:
It would be best to use `lib.versionAtLeast buildVersion "…"` to allow for bisecting and newer stable versions to switch as well.

Reply:
I checked and it was renamed in 4153.